content/docs/setup/kubernetes/install/multicluster/gateways/index.md

```diff
@@ -75,6 +76,7 @@ on **each** Kubernetes cluster.
     * Create a Kubernetes secret for your generated CA certificates using a command similar to the following. See [Certificate Authority (CA) certificates](/docs/tasks/security/plugin-ca-cert/#plugging-in-the-existing-certificate-and-key) for more details.
 
         {{< text bash >}}
+        $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin --user="$(gcloud config get-value core/account)"
```

Comment:
@lei-tang this can't be added here because it is using a google-specific command (`gcloud config ...`). If it is only needed for GCP, then there should be a warning icon telling GCP users to run this extra command. These instructions need to be platform-independent.

Reply:
https://github.com/istio/istio.io/pull/3959.